Northstar.Custom/mod/scripts/vscripts/sh_damage_types.nut

```diff
@@ -0,0 +1,776 @@
+
+		[ eDamageSourceId.prowler_melee ] 							= "#DEATH_PROWLER_MELEE",
+		[ eDamageSourceId.super_spectre_melee ] 					= "#DEATH_SUPER_SPECTRE",
+		[ eDamageSourceId.grunt_melee ] 							= "#DEATH_GRUNT_MELEE",
+		[ eDamageSourceId.spectre_melee ] 							= "#DEATH_SPECTRE_MELEE",
+		[ eDamageSourceId.eviscerate ]	 							= "#DEATH_EVISCERATE",
+		[ eDamageSourceId.wall_smash ] 								= "#DEATH_WALL_SMASH",
+		[ eDamageSourceId.ai_turret ] 								= "#DEATH_TURRET",
+		[ eDamageSourceId.team_switch ] 							= "#DEATH_TEAM_CHANGE",
+		[ eDamageSourceId.rocket ] 									= "#DEATH_ROCKET",
+		[ eDamageSourceId.titan_explosion ] 						= "#DEATH_TITAN_EXPLOSION",
+		[ eDamageSourceId.evac_dropship_explosion ] 				= "#DEATH_EVAC_DROPSHIP_EXPLOSION",
+		[ eDamageSourceId.flash_surge ] 							= "#DEATH_FLASH_SURGE",
+		[ eDamageSourceId.molotov ] 								= "#DEATH_MOLOTOV",
+		[ eDamageSourceId.sticky_time_bomb ] 						= "#DEATH_STICKY_TIME_BOMB",
+		[ eDamageSourceId.vortex_grenade ] 							= "#DEATH_VORTEX_GRENADE",
+		[ eDamageSourceId.droppod_impact ] 							= "#DEATH_DROPPOD_CRUSH",
+		[ eDamageSourceId.ai_turret_explosion ] 					= "#DEATH_TURRET_EXPLOSION",
+		[ eDamageSourceId.rodeo_trap ] 								= "#DEATH_RODEO_TRAP",
+		[ eDamageSourceId.round_end ] 								= "#DEATH_ROUND_END",
+		[ eDamageSourceId.burn ]	 								= "#DEATH_BURN",
+		[ eDamageSourceId.mind_crime ]								= "Mind Crime",
+		[ eDamageSourceId.charge_ball ]								= "Charge Ball",
+		[ eDamageSourceId.mp_titanweapon_rocketeer_missile ]		= "Rocketeer Missile",
+		[ eDamageSourceId.core_overload ]							= "#DEATH_CORE_OVERLOAD",
+		[ eDamageSourceId.mp_weapon_arc_trap ]						= "#WPN_ARC_TRAP",
+
+
+		[ eDamageSourceId.mp_turretweapon_sentry ] 					= "#WPN_SENTRY_TURRET",
+		[ eDamageSourceId.mp_turretweapon_blaster ] 				= "#WPN_BLASTER_TURRET",
+		[ eDamageSourceId.mp_turretweapon_rockets ] 				= "#WPN_ROCKET_TURRET",
+		[ eDamageSourceId.mp_turretweapon_plasma ]	 				= "#WPN_PLASMA_TURRET",
+
+		[ eDamageSourceId.bubble_shield ] 							= "#DEATH_BUBBLE_SHIELD",
+		[ eDamageSourceId.sticky_explosive ] 						= "#DEATH_STICKY_EXPLOSIVE",
+		[ eDamageSourceId.titan_grapple ] 							= "#DEATH_TITAN_GRAPPLE",
+
+		[ eDamageSourceId.satellite_strike ] 						= "#DEATH_SATELLITE_STRIKE",
+
+		[ eDamageSourceId.mp_titanweapon_meteor ] 					= "#WPN_TITAN_METEOR",
+		[ eDamageSourceId.mp_titanweapon_meteor_thermite ]			= "#WPN_TITAN_METEOR",
+		[ eDamageSourceId.mp_titanweapon_meteor_thermite_charged ]	= "Thermite Meteor",
+		[ eDamageSourceId.mp_titanweapon_flame_ring ]				= "Flame Wreath",
+
+		// Instant death. Show no percentages on death recap.
+		[ eDamageSourceId.fall ]		 							= "#DEATH_FALL",
+		 //Todo: Rename eDamageSourceId.splat with a more appropriate name. This damage type was used for enviornmental damage, but it was for eject killing pilots if they were near a ceiling. I've changed the localized string to "Enviornment Damage", but this will cause confusion in the future.
```

Comment:
Is this TODO something you still want to work on? ^^

Reply:
Isn't this Respawn's TODO?

Reply:
Respawn having unfinished code, smh